app/models/admin/digitization_queue_item.rb

```diff
@@ -23,6 +23,7 @@ def validate_each(record, attribute, value)
   COLLECTING_AREAS = %w{archives rare_books modern_library museum}
 
   validates :collecting_area, inclusion: { in: COLLECTING_AREAS }
+  validates :collecting_area, presence: true
```

Comment:
why this change, no longer making sure it is in the allow list of allowed collections?

Reply:
This was a mistake - thanks for catching it!